src/components/DecimalCapacity/index.tsx

```diff
@@ -32,7 +32,13 @@ export default ({
 
   return (
     <DecimalPanel>
-      <span>{integer}</span>
+      <span
+        style={{
+          color: color || 'rgba(0, 0, 0, 0.85)',
```

Comment:
We're trying to make the CSS clearer than before, so the inline style can be moved to an independent file